test/modules/linear_boltzmann_solvers/transport_steady/reed_balance.lua

```diff
@@ -0,0 +1,100 @@
+-- Standard Reed 1D 1-group problem
+
+-- Create Mesh
+widths = {2., 1., 2., 1., 2.}
+nrefs  = {200,200,200,200,200}
+
+Nmat = #widths
+
+nodes = {}
+counter = 1
+nodes[counter] = 0.
+for imat = 1, Nmat do
+    dx = widths[imat]/nrefs[imat]
+    for i = 1, nrefs[imat] do
+        counter = counter + 1
+        nodes[counter] = nodes[counter-1] + dx
+    end
+end
+
+meshgen = mesh.OrthogonalMeshGenerator.Create({ node_sets = {nodes} })
+mesh.MeshGenerator.Execute(meshgen)
+
+-- Set Material IDs
+z_min = 0.0
+z_max = widths[1]
+for imat = 1, Nmat do
+    z_max = z_min + widths[imat]
+    log.Log(LOG_0, 'imat='..imat..', zmin='..z_min..', zmax='..z_max)
+    lv = logvol.RPPLogicalVolume.Create(
+                {
+                    infx=true,
+                    infy=true,
+                    zmin=z_min, zmax=z_max
+                }
+    )
+    mesh.SetMaterialIDFromLogicalVolume(lv, imat-1)
+    z_min = z_max
+end
+
+-- Create materials
+mat_names = { "AbsoSrc", "Abso", "Void", "ScatSrc", "Scat"}
+materials = {}
+for imat = 1, Nmat do
+    materials[imat] = mat.AddMaterial(mat_names[imat])
+end
+
+-- Add cross sections to materials
+total = { 50., 5., 0., 1. , 1.  }
+c     = {  0., 0., 0., 0.9, 0.9 }
+for imat = 1, Nmat do
+    mat.AddProperty(materials[imat], TRANSPORT_XSECTIONS)
+    mat.SetProperty(materials[imat], TRANSPORT_XSECTIONS, SIMPLE_ONE_GROUP, total[imat], c[imat])
+end
+
+-- Create sources in 1st and 4th materials
+mat.AddProperty(materials[1], ISOTROPIC_MG_SOURCE)
+mat.SetProperty(materials[1], ISOTROPIC_MG_SOURCE, FROM_ARRAY, {50.})
+
+mat.AddProperty(materials[4], ISOTROPIC_MG_SOURCE)
+mat.SetProperty(materials[4], ISOTROPIC_MG_SOURCE, FROM_ARRAY, {1.})
+
+-- Angular Quadrature
+gl_quad = aquad.CreateProductQuadrature(GAUSS_LEGENDRE,64)
```

Comment:
Missing space after `,`

Reply:
I used Daryl's lua-format config file and used the LuaFormatter he suggested